docs/about/infrastructure/aws/how_to_use.md

```diff
@@ -0,0 +1,69 @@
+# How to use AWS in our projects
+
+## Getting credentials
+
+We try to use the [principle of least privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege). 
+That means that when you get credentials to access our AWS account, we will try to limit your 
+privileges as much as possible, so that you can still do the things you need to do with your 
+AWS account. This ensures that you don't accidentally do things you don't need to do that 
+can affect others (e.g., deleting data you are not working with, or starting machines when you 
+don't need to do that). 
+
+### Compromised credentials
+
+Compromised credentials are credentials that may have been exposed to people who should not have them. 
+One (scarily common) way in which credentials can get compromised is if they are somehow stored in 
+code and get committed to a repository and pushed to GitHub. Please note that if you committed credentials 
+into the history of a git repo, they will still be there in the history of the repo, 
+*even if you delete them and commit the deletion*. This means that if you committed the credentials, and you 
+are not sure how to remove them from the repo history, do not push that repo to GitHub, and ask for 
+someone to help.
+
+We hope to never have compromised credentials, but if you have reason to believe that your credentials were 
+compromised (e.g., you accidentally pushed them to GitHub), please let us know as soon as possible, so that 
+we can mitigate the potential harm of this.
+
+### Multi-factor authentication
+
+If your credentials allow you to log into the AWS console, you *must* use multi-factor 
+authentication. MFA means that in addition to entering your password when logging in, 
+another form of authentication will be required to get access to the console. For example, 
+a cell-phone app that provides you with a token. 
+
+## Rules of thumb for resource usage
+
+### Always tag resources with a "project" tag
+
+Tagging resources helps us track how much resources are being used in what project.
+Tagging is easy if you are using the console. Whenever starting a resource (e.g., 
+EC2 instance, or S3 bucket, you will have the option to tag it through the console).
+
+We currently have the following tags:
```

Comment:
Perhaps we should have another tag to indicate that a user is testing something out before it becomes a bigger project. Maybe "sandbox" or "dev" or something like that.